testing/test_full.py

```diff
@@ -3160,3 +3159,49 @@ def test_generic_data_split():
     sanitize_outputDir()
 
     print("passed")
+
+
+def test_gandlf_logging(capsys):
```

Comment:
I suggest renaming this so that it gets invoked by the tests [ref].

```suggestion
def test_generic_logging(capsys):
```